### ISSUE TITLE
All incoming non-redphone calls go to voicemail

### ISSUE THREAD

**Issue by kb100** [WhisperSystems/RedPhone#400]
This may or may not be related to https://github.com/WhisperSystems/RedPhone/issues/386
I also have a Galaxy S4 and all my incoming non-redphone calls go to voicemail. I'm not sure when this started happening since no missed call notification is ever displayed. Uninstalling redphone fixes the problem.

**Comment by 2-4601** [comment]
Have you tried disabling the _Prompt for call upgrade_ setting?

**Comment by kb100** [comment]
Yes, I have. I originally posted a bug report about not getting calls and it seemed like the Promt for call upgrade setting was the culprit. In that case I was still getting calls but they weren't ringing. I disabled the option and (thought) the problem was fixed. However, some months after I notice that calls are not even registering. No ring, nothing, even with the Prompt for call upgrade setting turned off. I don't know when the behavior started since I don't receive calls on that number very often and no one inquired as to why I wasn't responding to calls until this week.
